tests/test_pipreqs.py

```diff
@@ -322,6 +326,23 @@ def test_clean_with_imports_to_clean(self):
             data = f.read().lower()
             self.assertTrue(cleaned_module not in data)
 
+    def test_compare_modules(self):
+        tests = [
```

Comment:
tests is not best name for this, what do you think about something like test_cases, test_parameters or test_data?